src/shogun/multiclass/LaRank.cpp

```diff
@@ -588,7 +588,7 @@ CLaRank::CLaRank (): CMulticlassSVM(new CMulticlassOneVsRestStrategy()),
 	nb_seen_examples (0), nb_removed (0),
 	n_pro (0), n_rep (0), n_opt (0),
 	w_pro (1), w_rep (1), w_opt (1), y0 (0), m_dual (0),
-	batch_mode(true), step(0)
+	batch_mode(true), step(0), max_iteration(1000)
```

Comment:
The comments for CLaRank class or its constructors are almost empty. Should I add max_iteration to those comments? It might be more helpful to give just an overview of LaRank algorithm in those comments and then discuss the logistics behind ```max_iteration``` later in comments for relevant methods.

LaRank algorithm usually converges in very few iterations, and max_iteration, a relatively large number, should be rarely exceeded and may not need adjustments for most use. Should I still include the theoretic upper bound (the formula) to explain how to select the parameter?